tsconfig.json

```
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "experimentalDecorators": true,
+    "importHelpers": false,
+    "module": "commonjs",
```

```suggestion
    "module": "CommonJS",
```
The documentation does capitalize it. I'm guessing it's case-insensitive, but I figured it should match the docs